docs/snippets/numpydocExample.py

```diff
@@ -0,0 +1,297 @@
+#
+# This file is part of dm_dev_guide.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""Example Python module with Numpydoc-formatted docstrings.
+
+This module demonstrates documentation written according to LSST DM's guidelines for `Documenting Python
```

Comment:
I think these lines are too long for docstrings. I know we are still struggling with @brianv0's patch to flake8 but I really think our example docstrings should be under 80 characters if we are to have any ability to get people to use shorter lines.

Reply:
I guess it doesn't hurt to have the examples shorter. We can just remain silent on whether the regular-length docstring lines are required or not.

Reply:
Yes, we really don't want to go back and fix these examples later when flake8 finally does what we want.